wrappers/javascript/packages/anoncreds-nodejs/src/NodeJSAnoncreds.ts

```diff
@@ -617,6 +627,380 @@ export class NodeJSAnoncreds implements Anoncreds {
     return new ObjectHandle(handleReturnPointer<number>(ret))
   }
 
+  public createW3CCredentialOffer(options: {
```

Comment:
```suggestion
  public createW3cCredentialOffer(options: {
```

To not have confusion in casing, we always only capitalize the first letter of a word (and W3c is one word in this case)